sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/util/LatencyRecordingHttpRequestInitializer.java

```diff
@@ -23,14 +23,26 @@
 import com.google.api.client.http.HttpResponse;
 import com.google.api.client.http.HttpResponseInterceptor;
 import java.io.IOException;
-import org.apache.beam.sdk.util.Histogram;
+import org.apache.beam.sdk.metrics.Histogram;
+import org.apache.beam.sdk.metrics.Metrics;
+import org.apache.beam.sdk.util.HistogramData;
 
 /** HttpRequestInitializer for recording request to response latency of Http-based API calls. */
 public class LatencyRecordingHttpRequestInitializer implements HttpRequestInitializer {
+  public static final String HISTOGRAM_NAME = "latency_ms";
```

Comment:
Would you please use a specific name for the metric where it is instantiated

See the URN defined here:
API_REQUEST_LATENCIES
urn = beam:metric:io:api_request_latencies:v1
https://s.apache.org/beam-gcp-debuggability

For Labels:
for SERVICE_NAME use "BigQuery", METHOD_NAME you can use "BigQueryBatchWrite" and "BigQueryBatchRead". Which is just referring to API calls which read or write elements in batches.

Additional Labels (Populate these if they are available, if not leave it for follow up work for me):
BIGQUERY_PROJECT_ID
BIGQUERY_DATASET
BIGQUERY_TABLE
BIGQUERY_VIEW
BIGQUERY_QUERY_NAME- user provided query name
RESOURCE - combination of the above (See my python PR linked below for reference)

https://github.com/apache/beam/pull/13217/files#diff-5427a5d3887eb695cefde082c58575a2372972996b547d55961abeb4f7bc3debR576

Reply:
Done.